### PR TITLE
[merged] This series of pull requests moves the CLI option handling out of atomic into submodules.

### PR DESCRIPTION
**Pull request by rhatdan**
Simplify/shorten atomic and Atomic/atomic.py by moving each atomic `verb` to its own submodule.

### CODE REVIEW

**Comment by rhatdan** [comment]
@cgwalters @giuseppe @baude @shishir-a412ed PTAL

**Comment by cgwalters** [comment]
LGTM.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@jlebon @cgwalters Is the test suite hung up?

**Comment by cgwalters** [comment]
@rh-atomic-bot retry

**Comment by baude** [comment]
@jlebon does output.txt no longer include the log from the actual tests?

**Comment by baude** [comment]
@jlebon disregard previous question ... i see how to get to it.

File: Atomic/uninstall.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+def cli(subparser):
+    # atomic uninstall
+    uninstallp = subparser.add_parser(
+        "uninstall", help=_("execute container image uninstall method"),
+        epilog="atomic uninstall attempts to read the LABEL UNINSTALL "
+        "field in the image, if it does not exist atomic will "
+        "remove the image from your machine.  You could add a "
+        "LABEL UNINSTALL command to your Dockerfile like: 'LABEL "
+        "UNINSTALL %s'" % Uninstall.print_uninstall())
+    uninstallp.set_defaults(func='uninstall')
```
Comment (giuseppe) [inline Atomic/uninstall.py]: it misses: `_class=Uninstall`

**Comment by rhatdan** [comment]
I found three other fixes _class=Update _class=Uninstall and _class=Storage, which I fixed.  I can look at this again in the morning.

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**